dissect/target/plugins/os/unix/_os.py

```diff
@@ -200,6 +200,22 @@ def _add_mounts(self) -> None:
                     self.target.fs.mount(mount_point, volume.fs)
 
     def _parse_os_release(self, glob: Optional[str] = None) -> dict[str, str]:
+        """Parse Unix /etc/*-release files.
+
+        Not all release files are equal. Generally speaking release files are
+        either key=value files or contain just one line.
+
+        Examples of key=value pair structured release files are:
+        - /etc/os-release
+        - /usr/lib/os-release
+        - /etc/lsb-release
+
+        Examples of sparse release files:
```

Comment:
Could some tests be added for these sparse release files?

Reply:
`/etc/fedora-release` and `/etc/centos-release` are tested in the `test_unix_version_detection_short` function.

Reply:
Is this really a sparse release file? Since the explanation hints that these files should be one line "Generally speaking these files are either key=value files **_or contain just one line_**".

Reply:
Yes these files contain just one line:
```
$ docker run -it --rm centos:latest /bin/bash
[root@92e64b0491e9 /]# cat /etc/centos-release
CentOS Linux release 8.4.2105
[root@92e64b0491e9 /]#

$ docker run -it --rm fedora:latest /bin/bash
[root@c1ff53ed865b /]# cat /etc/fedora-release 
Fedora release 37 (Thirty Seven)
[root@c1ff53ed865b /]#